inference-engine/src/vpu/graph_transformer/src/middleend/pass_manager.cpp

```diff
@@ -172,6 +172,9 @@ PassSet::Ptr PassManager::buildMiddleEnd() {
         ADD_PASS(swapConcatAndHwOps);
         ADD_DUMP_PASS("swapConcatAndHwOps");
 
+        ADD_PASS(reshapeBeforeConvTiling);
+        ADD_DUMP_PASS("reshapeBeforeConvTiling");
+
```

Comment:
Text does not seem to be correct.

```
convolution stage that will trigger the "hwConvTiling" pass
```

stage does not trigger pass. 

I would advise:

```
"reshapeBeforeConvTiling" pass changes geometry of convolution stages in order to get more efficient HW tiling using reshape stages.
```

```
This pass changes convolution input and output, width and height
```

Please review that part. I think you can just remove this.

```
where changes original stage attributes
```

Could you please explain what did you mean there?

Reply:
fixed

Reply:
Fixed

Reply:
> where changes stage "origConvOutput" attribute

It's still unnatural English to me.. `where changes ...` - who performs this change?

Do you want to say that pass should be located before `adjustDataBatch` because it modifies `origConvOutput` attribute which is used in `adjustDataBatch` and `adjustDataBatch` should be able to see the latest version?

Reply:
New comment:
"reshapeBeforeConvTiling" pass changes geometry of convolution stages in order to get more efficient HW tiling (pass "hwConvTiling") using reshape stages.

Pass should be located before "adjustDataBatch" because "reshapeBeforeConvTiling"
changes geometry of convolution. "adjustDataBatch" modifies stage "origConvOutput"
attribute to the current convolution output at the time of execution of "adjustDataBatch" pass.
"origConvOutput" attribute is used in "hwConvTiling" and
"hwConvTiling" should be able to see the latest version of this attribute.